packages/message-box-react/documentation/Example.tsx

```diff
@@ -18,12 +18,22 @@ const getTypeOfBox = (typeofBox?: string) => {
 };
 
 export const Example: React.FC<ExampleComponentProps> = ({ boolValues, choiceValues }) => {
+    const [dismissed, setDismissed] = useState(false);
     const C = getTypeOfBox(choiceValues ? choiceValues["Type"] : "");
+    const dismissAction =
+        boolValues && boolValues["Avvisbar"]
```

Comment:
Er det ikke mulig å bruke optional-chaining her? boolValues?.["Avvisbar"] ? { ... } : undefined

Reply:
Joa, er nok det nå! Bare fortsetter med samme pattern som vi har brukt alle de andre stedene for å være konsekvent 😊